internal/provider/provider.go

```diff
@@ -166,6 +169,15 @@ func (p *PowerPlatformProvider) Schema(ctx context.Context, req provider.SchemaR
 				MarkdownDescription: "Flag to indicate whether to opt out of telemetry. Default is `false`",
 				Optional:            true,
 			},
+			"partner_id": schema.StringAttribute{
+				MarkdownDescription: "The GUID of the partner for Customer Usage Attribution (CUA).",
+				Optional:            true,
+				CustomType:          customtypes.UUIDType{},
+			},
+			"disable_terraform_partner_id": schema.BoolAttribute{
+				MarkdownDescription: "Disable sending the default Terraform partner ID when no custom partner_id is provided. Default is `false`",
```

Comment:
I think this should have a validation, that it should be only used together with `partner_id`

Reply:
it can be used without partner_id.  see description above